datadotworld/models/dataset.py

```diff
@@ -21,12 +21,14 @@
 import os
 import warnings
 import io
-from collections import OrderedDict
+try:
+    from collections.abc import OrderedDict
+except ImportError:
+    from collections import OrderedDict
```

Comment:
`collections.abc` is the proper package in Python 3 and simply importing `collections` is deprecated and will return an error as of Python 3.8.